src/chatbot_utils.py

```diff
@@ -298,9 +302,9 @@ def display_chat_interface(self):
                                                            "personal information or copyrighted material.")
                         st.write(f"{using_text} **{os.getenv('OPENAI_MODEL')}**. {remember_text}")
 
-                st.markdown(session_state['_']("Edit System Prompt"), help=session_state['_'](
-                        "The system prompt is transmitted with each of your entries. "
-                        "You can edit the system prompt in the text field."))
+                # st.markdown(session_state['_']("Edit System Prompt"), help=session_state['_'](
```

Comment:
What is the reason of commenting these lines? If we don't need them, we can delete them.

Reply:
Updated (commented code was deleted)